test/verify/test_multinomial.cpp

```diff
@@ -64,3 +64,6 @@ template struct test_multinomial<migraphx::shape::float_type>;
 template struct test_multinomial<migraphx::shape::half_type>;
 // TODO This fails, need to figure out why
```

Comment:
Is there an issue open for this or are we tracking this in any way?

Reply:
yup, https://github.com/ROCm/AMDMIGraphX/issues/2516